democracy_club/templates/hermes/post_detail.html

```diff
@@ -32,6 +32,12 @@ <h1>{{ post.subject }}</h1>
                     {% endfor %}
                 </address>
                 on <time pubdate datetime="{{post.modified_on|date:"c"}}" title="{{post.created_on|date:"jS E Y"}}">{{post.created_on|date:"jS E Y"}}</time>
+                {% if post.tags %}
+                    <span aria-hidden="true">🏷️</span>
+                    {% for tag in post.tags %}
+                        <a href="{% url 'hermes_post_list_by_tag' tag %}">{{ tag }}</a>{% if not forloop.last %}, {% endif %}
+                    {% endfor %}
```

Comment:
Can you render tags as a `ul` & `li` list? And add some CSS to show them inline rather than as a list? This will help screen readers read them out as tags.

Reply:
Fixed. Ran Lighthouse and the page scored 100%.